geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/stats/Stat.scala

```diff
@@ -112,6 +114,34 @@ trait Stat {
  */
 object Stat {
 
+  val DoubleSerializer = new JsonSerializer[jDouble]() {
+    def serialize(d: jDouble, t: Type, jsc: JsonSerializationContext): JsonElement = d match {
+      /* NaN check, use null to mirror existing behavior for missing/invalid values */
+      case d if jDouble.isNaN(d) => JsonNull.INSTANCE
+      case d if d == jDouble.NEGATIVE_INFINITY => new JsonPrimitive("Infinity")
+      case d if d == jDouble.POSITIVE_INFINITY => new JsonPrimitive("+Infinity")
+      case _ => new JsonPrimitive(d)
+    }
+  }
+
+  val FloatSerializer = new JsonSerializer[jFloat]() {
+    def serialize(f: jFloat, t: Type, jsc: JsonSerializationContext): JsonElement = f match {
+      /* NaN check, use null to mirror existing behavior for missing/invalid values */
+      case f if jFloat.isNaN(f) => JsonNull.INSTANCE
+      case f if f == jFloat.NEGATIVE_INFINITY => new JsonPrimitive("Infinity")
+      case f if f == jFloat.POSITIVE_INFINITY => new JsonPrimitive("+Infinity")
+      case _ => new JsonPrimitive(f)
+    }
+  }
+
+  val JSON: Gson = new GsonBuilder()
```

Comment:
I'd verified this, from the GSON javadoc ["This is the main class for using Gson. Gson is typically used by first constructing a Gson instance and then invoking toJson(Object) or fromJson(String, Class) methods on it. Gson instances are Thread-safe so you can reuse them freely across multiple threads."](https://google.github.io/gson/apidocs/com/google/gson/Gson.html)